Library/Homebrew/linkage_checker.rb

```diff
@@ -83,6 +86,12 @@ def check_undeclared_deps
       next true if Formula[name].bin.directory?
       @brewed_dylibs.keys.map { |x| x.split("/").last }.include?(name)
     end
+    missing = []
+    @broken_dylibs.each do |str|
+      next unless str.start_with? "#{HOMEBREW_PREFIX}/opt"
+      missing << str.sub("#{HOMEBREW_PREFIX}/opt/", "").split("/")[0]
+    end
+    unnecessary_deps -= missing
```

Comment:
It would be good to maybe output these `missing` in some form?

Reply:
But missing deps can already be found with `brew missing`...

Reply:
I can convert `broken_dylibs` to Hash to make `Missing libraries:` output similar to that of `Homebrew libraries:`, that is:

<pre>
$ brew linkage tor
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libz.1.dylib
Homebrew libraries:
  /usr/local/opt/libevent/lib/libevent-2.1.6.dylib (libevent)
Missing libraries:
  /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib <b>(openssl)</b>
  /usr/local/opt/openssl/lib/libssl.1.0.0.dylib <b>(openssl)</b>
</pre>

Would that work?

Reply:
That seems like a good call 👍